src/components/organisms/Footer/Footer.test.tsx

```diff
@@ -0,0 +1,12 @@
+import { render, screen } from '@testing-library/react';
+
+import Footer from './Footer';
+
+describe('Footer', () => {
+  it('renders correctly and show current year', () => {
```

Comment:
I suppose it would be also good to validate the following footer elements:
- All social media links
- Copyrights
- Contact details